netutil/interface.go

```diff
@@ -79,7 +79,10 @@ func NextBundleInterface(t *testing.T, dut *ondatra.DUTDevice) string {
 	t.Helper()
 	batch := dut.Telemetry().NewBatch()
 	dut.Telemetry().InterfaceAny().Aggregation().Batch(t, batch)
-	bundleIntfs := batch.Lookup(t).Val(t).Interface
+	bundleIntfs := make(map[string]*telemetry.Interface)
+	if bundles:=batch.Lookup(t); bundles!=nil {
+		bundleIntfs = bundles.Val(t).Interface
+	}
 	name, err := nextBundleInterface(dut.Vendor(), bundleIntfs)
```

Comment:
please change nextBundleInterface to accept a *QualifiedDevice as its second argument, so you can pass in the result of batch.Lookup(t). That then allows you to easily add a unit test for this behavior to interface_test.go. Same with nextVLANInterface

Reply:
thanks, done